.github/workflows/zero-downtime-tests.yml

```
@@ -1,13 +1,456 @@
-name: Zero-Downtime Tests
+name: ZeroDowntime Tests
 on:
-  workflow_dispatch:
+  workflow_dispatch: # run integration tests only when triggered manually
+    inputs:
+      branch:
+        description: 'Take CI build artifacts from branch (e.g., master, release-x.y.z)'
+        required: true
+        default: 'master'
+      upgradeTo:
+        description: 'Helm chart path to run upgrade command'
```

```suggestion
        description: 'The helm upgrade command will be run with this version'
```
just a suggestion...